### DISCUSSION TITLE
compiling firmware but can't home.

### DISCUSSION THREAD

**Discussion by cruzer619** [Jyers/Marlin#1028] [Q&A] [answered]
ok so today I took the dive in learning how to compile.  guess you can teach old dogs new tricks.. :P I was able to compile vanilla marlin for my ender 3 pro.  And everything works..  So then I went ahead and compiled jyers for my Ender 3V2.  compiles no errors but for the life of me, my printer won't home.  If I flash my favorite binary works just fine..  i'm using a stock bltouch 3x3 hs template..  change it to 5x5.. only change I made.  but the darn thing won't home.. what am I doing wrong.

**Comment by smooreace** [comment, marked as answer]
You are likely running a 4.2.7 mainboard?

**Reply by cruzer619** [reply]
yup that was it.. I didn't change motherboard to "MOTHERBOARD BOARD_CREALITY_V427"  from "MOTHERBOARD BOARD_CREALITY_V4"
